services/reviewService/components/Review.jsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable react/prop-types */
+// this component will render individual reviews
+import React from 'react';
```

Comment:
Did SSR require a refactor on the frontend as well? If so, was it a complete refactor or a minor refactor?

Reply:
Not much. I just chose to clean up some extra stuff that was either causing errors or just made more sense it me. Using the renderToString method should work with any react component you throw in it.